our_name_is_platform_engineering_team.md

```diff
@@ -0,0 +1,87 @@
+
+* the activity this team is doing
+* the group of people doing that activity
+* the collective output/work of the team
+
+The DevOps Toolkit video ["Devops is Dead! Log Live Platform Engineering! Did We
+get Confused?"](https://www.youtube.com/watch?v=9_v77YiSGEY) (especially the
+secion at [12:32](https://youtu.be/9_v77YiSGEY?t=752)) inspired the creation of
+this team, and in the video this work is called "Platform Engineering".
+
+It was imporant that the name describe the work that we are doing, and that it
+not be distracting when talked about in meetings. That ruled out names like
+
+* cuddly barnacle (an in-joke about github automatic repository naming)
+* The Shipping Industry (a reasonable metaphor, but hard to take seriously)
+* ShipIt (too indirect and too cute)
+
+"The Deploy System" was considered because it describes more-or-less what we're
+doing and it would draw attention to the similarity between this team and The
+Design System. We did not choose this name because:
+
+1. It would look too much like The Design System, and be abbreviated the same
+   way: "DS"
+1. We have even more projects that start with "d" like "Deep Blue" and "Deep
+   Blue Data" and "Deep Blue Docs".
+1. The word "deployment" is a particular Kubernetes term, so its use here as a
+   term for everything that this group does is confusing.
+1. This group's scope may include more than "deploying" systems, although the
+   Design System is also more expansive than the name suggests so this may be
+   not be a big problem.
+
+"The Platform" was considered as the name for the "collective output/work of the
+team." We did not choose this name because:
+
+1. "The Platform" has a connotation of being "the one and only way of doing
+   something"
+1. There is a history in LIT of the phrase "The Platform", and we do not want to
+   draw a connection between this group and those ideas
+1. The idea of "The Platform" being an expansive term for "the collection of
+   ways we put applications out into the world" is not an industry-wide accepted
+   idea
+1. In general the term "platform" for tech products and service can refer to
+   many different things, most of which is not what we're doing on this team.
+
+## Decision
+
+* the activity this team is doing: *Platform Engineering*
+* the group of people doing that activity: *The Platform Engineering Team*
+* the collective output/work of the team: *The work of The Platform Engineering
+  Team* i.e. there is no official name
+
+We chose *Platform Engineering* because that is the industry term for the work
+this group is doing. Adding the word "team" to describe the group of people is
+straight-forward to understand.
+
+We are choosing **not** to give a name to the
+collective output because it doesn't need an official branded name. People will
+know to look for the work of the team under the name "Platform Engineering". We
+can talk about the work without referring to the collective output of the team.
```

Comment:
I think you've covered almost everything so far. However, I think it is important to indicate the combined spirit of collaboration and improvement. I think we will and should be embedded with teams from time to time, working on a local solution as well as capturing it so others can learn from it or extracting "something", aside from knowledge, that can be reused more directly.

I definitely agree that we should not seek a pithy name for it, but I do think we can draw a somewhat more descriptive, yet concise, box around what we think that work includes. I suppose this could be covered (maybe more effectively) in a charter but, wherever that happens, I don't want us to skip it.